packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -320,6 +340,38 @@ export default class InteractSubmode extends Component<Signature> {
     }
   });
 
+  private runCommand = restartableTask(
+    async (
+      card: CardDef,
+      skillCardId: string,
+      skillCardRealm: URL,
+      message: string,
+    ) => {
+      try {
+        let [commandCard] = await this.cardService.search(
+          { filter: { eq: { id: skillCardId } } },
+          skillCardRealm,
+        );
+        if (!commandCard) {
+          throw new Error(
+            `Could not find card "${skillCardId}" in realm "${skillCardRealm}"`,
+          );
+        }
+        let newRoomId = await this.matrixService.createRoom(`New AI chat`, [
+          aiBotUsername,
+        ]);
+        await this.matrixService.sendMessage(
+          newRoomId,
+          message,
+          [card],
+          [commandCard],
+        );
+      } catch (e) {
```

Comment:
Do we need a try/catch if we just catch then rethrow the error?

Reply:
Nope, removed it. I used some of your ideas from your branch for cs-6995 for sending a message with the command. I see you're using command cards and tools there, feel free to update this as you see fit. Also added a disabled "Generate App Now" button on the PRD card if you want to add your code there.

Reply:
Ah and the skill card I have for product requirements is very basic, we can improve that too.